ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -585,7 +596,7 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         let (actualStyle, actualItem) = actualStyleAndItem(for: navigationItem)
         style = actualStyle
         updateColors(for: actualItem)
-        showsLargeTitle = navigationItem.usesLargeTitle
+        usesAvatarTitleView = navigationItem.titleStyle != .system
```

Comment:
This logic should probably be
```
let titleStyle = navigationItem.titleStyle
usesAvatarTitleView = titleStyle == .leading || titleStyle == .largeLeading
```
to make it more resilient to future style additions.

Reply:
We might as well bake this into a computed property of `NavigationBar.TitleStyle` so we don't need to have this verbosity all over the place.